lib/ri_cal/component/calendar.rb

```diff
@@ -200,18 +200,18 @@ def utf8_safe_split(string, n)
                 n = n - 1
                 before = string[0, n]
                 after = string[n..-1]
-              end      
+              end
               [before, after.empty? ? nil : after]
             end
           end
         end
 
         def fold(string) #:nodoc:
           line, remainder = *utf8_safe_split(string, 73)
-          stream.puts(line)
+          stream.puts("#{line}\r")
```

Comment:
Yep, or at least that's how I understand it. There is a _little_ bit of ambiguity in the [docs](http://ruby-doc.org/core-2.2.3/IO.html#method-i-puts) regarding the "record separator" though.